src/main/proto/handshake.proto

```diff
@@ -40,11 +40,13 @@ message ClientInitMsg {
 message UseAuthTypeMsg {
   AuthTypeEnum auth_type = 1;
   bytes salt = 2;
+  bytes iv = 4;
```

Comment:
any reason what 4 is before 3?

Reply:
Just that conceptually the salt and IV belong together as encryption parameters and it makes sense for the challenge to come after, but my understanding of the reason why protobufs lets you specify the field order is that they're theoretically backwards compatible.
MapTool doesn't need to support that so I can't think of any problems with changing the order.